otri/filtering/filters/average_filter.py

```diff
@@ -0,0 +1,69 @@
+from ..filter import Filter, Stream, Collection
+from typing import Sequence, Mapping, Dict
+from numbers import Number
+
+
+class AverageFilter(Filter):
```

Comment:
Define inputs and outputs of filter in class specification, so it's easier and faster to link them together.